### PR TITLE
TANF Formula for tanf_countable_income

### LINKED ISSUE #368: Add TANF IL 3/4 earned income deduction
https://www.dhs.state.il.us/page.aspx?item=15868

>Compare the total countable monthly income to the Payment Level after deducting:
...
the 3/4 earned income deduction

### PR DESCRIPTION
**Pull request by yvvijay121**
# New formula for TANF countable income

- [x] Label field added
- [x] Documentation field added
- [x] Unit field added
- [x] Default value field added if relevant
- [x] Variable name follows conventions
- [x] Unit test(s) added

## What's changed

- Added formula to `tanf_countable_income` variable in `tanf.py`
- Added variable `tanf_total_gross_income` in `tanf.py`
- Fixes #368.

### CODE REVIEW

**Comment by nikhilwoodruff** [comment]
Thanks @yvvijay121 - so I think the tests are failing because the household in the test cases doesn't have members specified, causing the people in the spm unit to get their own households (which default to CA). Adding a person, and specifying that person in the `members` for each group (spm unit and household) should fix this.

File: openfisca_us/variables/hhs/tanf.py [hunk 1]
```diff
@@ -74,6 +74,24 @@ class tanf_countable_income(Variable):
...
+        tanf_gross_income = spm_unit("tanf_total_gross_income", period)
+        state = spm_unit.household("state_code_str", period)
+        earned_income_deduction = parameters(
+            period
+        ).hhs.tanf.earned_income_deduction
+        return tanf_gross_income * (1 - earned_income_deduction[state])
+
+
+class tanf_total_gross_income(Variable):
+    # will include both gross earned and unearned income: read https://www.dhs.state.il.us/page.aspx?item=15814
```
Comment (MaxGhenis) [inline openfisca_us/variables/hhs/tanf.py:87]: And move below - including gross + unearned is in the documentation
```suggestion
    reference: "https://www.dhs.state.il.us/page.aspx?item=15814"
```

File: openfisca_us/tests/policy/baseline/hhs/tanf/tanf_countable_income.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        age: 10
+    spm_units:
+      spm_unit:
+        members: [parent, child]
+        tanf_total_gross_income: 3500
+    households:
+      household:
+        state_code: IL
+  output:
+    tanf_countable_income: 3500 / 4
```
Comment (MaxGhenis) [inline openfisca_us/tests/policy/baseline/hhs/tanf/tanf_countable_income.yaml:18]: newline

**Review by MaxGhenis — APPROVED** [review]
good to go after this

**Pull request merged**